### PR TITLE
Enable Gemma3 multimodal model

### PR DESCRIPTION
**Pull request by jambayk**

### CODE REVIEW

File: src/torch_onnx_models/components/_attention_utils.py [hunk 1]
```diff
@@ -7,40 +7,37 @@
...
-        "attention_mask should be of shape (batch_size, total_length)"
-    )
+    assert input_ids.dim() == 2, "input_ids should be of shape (batch_size, query_length)"
+    assert attention_mask.dim() == 2, "attention_mask should be of shape (batch_size, total_length)"
+    query_length = input_ids.shape[-1]
     all_indices = attention_mask.cumsum(-1)
     kv_indices = torch.unsqueeze(all_indices, 1)
-    # should we make this not data dependent slicing?
-    # like q_indices = torch.arange(query_length, device=attention_mask.device)
+    # exporter must provide a non-empty past, otherwise torch export thinks query_length == kv_length
```
Comment (justinchuby) [inline src/torch_onnx_models/components/_attention_utils.py:34]: Is there a way for us to fix it?

Comment (jambayk) [reply src/torch_onnx_models/components/_attention_utils.py:34]: i think we can do remove the data-dependent slicing using range or something similar. but with the shape inference where it things seq_len = total_len, I am not sure how to fix it. even when we give dynamic shape for attention mask as total_len, it gets infered as seq_len if the past_len is 0 and attention mask has same shape as input_ids
